app/views/browse/build-config.html

```diff
@@ -254,7 +254,7 @@
                                 </div>
                                 <dt ng-if-start="buildConfig.spec.source.binary.asFile">Binary Input as File:</dt>
                                 <dd ng-if-end>{{buildConfig.spec.source.binary.asFile}}</dd>
-                                <div ng-if="buildConfig.spec.source.type == 'None'">
+                                <div ng-if="buildConfig.spec.source.type == 'None' && !(buildConfig | isJenkinsPipelineStrategy)">
```

Comment:
You need to use Git source and Jenkinsfile path if you do not embed the Jenkinsfile.